### ISSUE TITLE
Do some dimensions come for free?

### ISSUE THREAD

**Issue by BorBorBor** [malb/lattice-estimator#58]
Hi,

I've been trying to answer this question myself by digging through old commit messages (not recommended…) but I haven't been able to find the answer: does the estimator take into account Ducas' [ePrint 2017/999 Shortest Vector from Lattice Sieving: a Few Dimensions for Free](https://eprint.iacr.org/2017/999)? 

Maybe more general, is there an up-to-date list of which works are implemented/taken into account currently?

(apologies for reverse-"well actually I have more of a comment than a question"-ing the issue feature)

Cheers,
Bor

**Comment by malb** [comment]
Yup. This depends on the reduction cost model used and e.g. the "Kyber" one (and derived ones) uses d4f: https://github.com/malb/lattice-estimator/blob/main/estimator/reduction.py#L701

The list of references is here: https://lattice-estimator.readthedocs.io/en/latest/references.html
